PetGame/PetGame/PetData.swift

```diff
@@ -95,16 +95,37 @@ class PetData{
     }
     
     func Feed() -> Void{
+        
+        if(isSameDate(dateOne: lastFed, dateTwo: NSDate())){
+            AchievementHandler.instance.PostAchievement(key_: "Feed 1", currentValue_:  (AchievementHandler.instance.Achievements["Feed 1"]?.currentValue)! + 1)
+            AchievementHandler.instance.PostAchievement(key_: "Feed 7", currentValue_:  (AchievementHandler.instance.Achievements["Feed 7"]?.currentValue)! + 1)
+            AchievementHandler.instance.PostAchievement(key_: "Feed 30", currentValue_:  (AchievementHandler.instance.Achievements["Feed 30"]?.currentValue)! + 1)
+        }
+        
         lastFed = NSDate()
         StoreData()
     }
     
     func Wash() -> Void{
+        
+        if(isSameDate(dateOne: lastWashed, dateTwo: NSDate())){
+            AchievementHandler.instance.PostAchievement(key_: "Wash 1", currentValue_:  (AchievementHandler.instance.Achievements["Wash 1"]?.currentValue)! + 1)
```

Comment:
Avoid using hardcoded string values, this will become more apparent to why its an issue if you ever have a project that needs to be localized in different languages but I realize with the classes current experiences this doesnt seem to be a big deal. But it is.
```
struct Constants {
     struct Achievements {
          static let wash1 = "Wash 1" // NSLocalizedString("Wash 1", "The name of the achievement for doing your first wash");
     }
}
```

Reply:
Then in your code you can use it like: 
```
AchievementHandler.instance.PostAchievement(key_: Constants.Achievements.wash1, currentValue_: ...
```